crates/api_models/src/admin.rs

```diff
@@ -93,10 +93,33 @@ pub struct MerchantAccountCreate {
     #[schema(value_type = Option<Object>,example = json!({"type": "single", "data": "signifyd"}))]
     pub frm_routing_algorithm: Option<serde_json::Value>,
 
-    /// The id of the organization to which the merchant belongs to
+    /// The id of the organization to which the merchant belongs to, if not passed an organization is created
     pub organization_id: Option<String>,
 }
 
+#[cfg(feature = "v2")]
+#[derive(Clone, Debug, Deserialize, ToSchema, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct MerchantAccountCreateV2 {
+    /// The identifier for the Merchant Account
+    #[schema(max_length = 64, example = "y3oqhf46pyzuxjbcn2giaqnb44")]
+    pub id: String,
+
+    /// Name of the Merchant Account
+    #[schema(value_type= Option<String>, example = "NewAge Retailer")]
+    pub merchant_name: Option<Secret<String>>,
+
+    /// Details about the merchant
+    pub merchant_details: Option<MerchantDetails>,
+
+    /// You can specify up to 50 keys, with key names up to 40 characters long and values up to 500 characters long. Metadata is useful for storing additional, structured information on an object.
```

Comment:
Are the 50 key and 40 character limits being enforced in code? If not, can we remove that from the docs (in both old and new request and response types)?